#158422834 User should be able to view and accept request for the ride offer he or she has given

LarryKarani opened the pull request:
**What does this PR do?**
This pull request create a  page where users can view and accept request for the ride offer they created 

**Description of Task to be completed?**
The page should contain a brief description of the person requesting the ride
The page should contain a photo of the person requesting the ride
The page should contain option buttons that allows the onwer of the ride offer to either accept or reject ride request.

**How the app should be tested**
Clone the repo to your local machine
Navigate to Ride My Way then to UI
open the offer file using any suitable browser (preferably chrome)

**Any background context you want to provide?**
The User Interface has no functionality.
Developed using HTML CSS

**What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/n/projects/2179558
 ID [#158422834]

**section of the page**

![reqt](https://user-images.githubusercontent.com/22986772/41663288-b5d03a76-74ab-11e8-85dd-4f5f06912b46.PNG)

LarryKarani committed:
Add a login page finishes[#158422013]

LarryKarani committed:
added signup.html and index.html

LarryKarani committed:
Merge branch 'master' into feature

LarryKarani committed:
Add The Ride offer view page
  - the page shows a list of all the available ride offers
  - the page has a details navigation button
  - the page has request ride offer navigation button
Adds The details view page
  -the page contains all the deatils partaining a certain ride offer
Edit style.css
  _change some features styles
Edit the login and signup form
  -added the required atribute to the input field
  finishes[#15822386]
  finishes[#158422553]

LarryKarani committed:
Merge branch 'feature' of https://github.com/larryTheGeek/RideMyWay into feature

LarryKarani committed:
modified rideoffers.html
 -add links to request.html to allow users to respond to ride offer

Add request.html
 - this page comprises of a form that allows users to respond to ride
   offers

LarryKarani committed:
modified the style.css by  removing comments
 - removed comments as advised in the PR

LarryKarani committed:
fixed a typo

LarryKarani committed:
Add offer.html that shows request to ride offer
  -offer.html allows users to view the request to the ride offer provided
  -the page contains an image and a brief detail about the user requesting
  a ride offer

SelaDanti commented:
Maybe You can add a slot for Friends ie when i want to book your ride for me an 2 other people

larryTheGeek commented:
good idea ill work on that

LarryKarani committed:
Add user, driver profile page and the history page
  -the profile pages provides details about the users
  -the history page provides details bout the rides offersd and those
  given

  finishes[#158422982]
  finishes[#158422707]

LarryKarani committed:
Add a profile page for both users and drive.
 the pages are linked to the details partaing the specific user
```add history page that show past user activities```
```add a landing page linked to all relevant landing pages```
```add separate acounts for each user ie driver and passanger```
finishes[#158422982] [#158422834]

LarryKarani committed:
fixed a typo

LarryKarani committed:
changed color and driver profile outlook

LarryKarani committed:
changed the UI pages by adding a nav to each page redesign the profile page of both user and driver, redesigned the view requst page
